LEGACY_SSE_EMULATION.md

```diff
@@ -158,26 +176,32 @@ Legacy SSE event will be the same
+    - Version2 `next_era_validator_weights` will be copied from Version2 `next_era_validator_weights`.
+    - Version1 `era_report` will be assembled from the Version2 `era_end.equivocators`, `era_end.rewards` and `era_end.inactive_validators` fields.
+    - If one of the `rewards` contains a reward that doesn't fit in a u64 (because Version2 has U512 type in rewards values) - the whole `era_end` **WILL BE OMITTED** from the legacy Version1 block (value None).
+    - Version2 field `next_era_gas_price` has no equivalent in Version1 and will be omitted.
+- `block.header.current_gas_price` this field only exists in Version2 and will be omitted from the Version1 block header.
+- `block.header.proposer` will be copied from Version2 to Version1 `block.body.proposer`.
+- other `block.header.*` fields will be copied from Version2 to Version1.
+- `block.body.deploy_hashes` will be based on Version2 `block.body.standard` transactions. Bear in mind, that only values of transactions of type `Deploy` will be copied to Version1 `block.body.deploy_hashes` array.
+- `block.body.transfer_hashes` will be based on Version2 `block.body.mint` transactions. Bear in mind, that only values of transactions of type `Deploy` will be copied to Version1 `block.body.transfer_hashes` array.
 
-- When the 2.x event emits a V2 block the following rules apply:
-
-  - `block_hash` will be copied from V2 to V1
-  - `block.block_hash` will be copied from V2 to V1
-  - `block.header.era_end`:
-    - if the era_end is a V1 variety - it will be copied
-    - if the era_end is a V2 variety:
-      - V2 `next_era_validator_weights` will be copied from V2 `next_era_validator_weights`
-      - V1 `era_report` will be assembled from V2 `era_end.equivocators`, `era_end.rewards` and `era_end.inactive_validators` fields
-      - IF one of the `rewards` contains a reward that doesn't fit in a u64 (because V2 has U512 type in rewards values) - the whole `era_end` **WILL BE OMITTED** from the legacy V1 block (value None)
-      - V2 field `next_era_gas_price` has no equivalent in V1 and will be omitted
-  - `block.header.current_gas_price` this field only exists in V2 and will be omitted from the V1 block header
-  - `block.header.proposer` will be copied from V2 to V1 `block.body.proposer`
-  - other `block.header.*` fields will be copied from V2 to V1
-  - `block.body.deploy_hashes` will be based on V2 `block.body.standard` transactions. Bear in mind, that only values of transactions of type `Deploy` will be copied to V1 `block.body.deploy_hashes` array
-  - `block.body.transfer_hashes` will be based on V2 `block.body.mint` transactions. Bear in mind, that only values of transactions of type `Deploy` will be copied to V1 `block.body.transfer_hashes` array.
-
-  An example of the above rules.
-  Input V2 BlockAdded:
+Here is an example mapping demonstrating the rules above:
+
+<!-- TODO Why do we have protocol_version: 1.0.0 for the Version2 block? This seems incorrect in this example. >
```

Comment:
The protocol number is just a value, it can be set to anything. If someone spins up a custom chain they don't have to follow the versioning that mainnet has. That said - if you think it's confusing, feel free to change it to 2.0.0.

Reply:
I've updated it to 2.0.0 in this example.